cli/core/pkg/pluginmanager/manager.go

```diff
@@ -488,6 +488,20 @@ func installOrUpgradePlugin(serverName string, p *plugin.Discovered, version str
 	descriptor.Discovery = p.Source
 	descriptor.DiscoveredRecommendedVersion = p.RecommendedVersion
 
+	if installTestPlugin {
+		log.Infof("Installing test plugin for '%v:%v'", p.Name, version)
+		b, err = p.Distribution.FetchTest(version, runtime.GOOS, runtime.GOARCH)
+		if err != nil {
+			log.Infof("Unable to install test plugin for '%v:%v'. Error: %v", p.Name, version, err)
```

Comment:
We just want to log if test plugin install errors out?

Reply:
Thanks for catching this. Updated.